src/main/java/uk/gov/hmcts/reform/et/syaapi/models/AcasCertificateRequest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.et.syaapi.models;
+
+import java.util.Arrays;
+
+/**
+ * Represents the content of a request sent to ACAS, containing the certificate numbers to be requesting certificate
+ * data for.
+ */
+public class AcasCertificateRequest {
+
+    private String[] certificateNumbers;
```

Comment:
Instead of String array, can we not use ArrayList of String?

Reply:
Agreed to keep as String array